### ISSUE TITLE
Special logic for scripts

### ISSUE THREAD

**Issue by ai** [lamartire/sharec#61]
It is very popular pattern to have `&&` list of commands in `package.json` scripts:

```json
  "test": "jest --coverage && eslint . && size-limit && yarn spell"
```

We can tread these scripts as lists: if locally one element changed we can ignore this element, but update others.

Of course, we should not hurry with this issue and do https://github.com/lamartire/sharec/issues/47 first

**Comment by lamartire** [comment]
Got it. Yeah, this is a good idea.

**Comment by lamartire** [comment]
@ai released at `2.5.0` :tada:
